src/accordion/accordion-section.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import Button from '../button/button.svelte';
+  import { createEventDispatcher } from 'svelte';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+
+  export let label = null;
+  export let open = false;
+
+  const selfControl = {
+    close() {
```

Comment:
So there is `close` and `toggle` but not `open`?

Reply:
This `selfControl` is a piece of internal API anyway, and only `toggle` is exposed as a slot prop, so no problem here. `open` is not used internally.